### PR TITLE
counted data scraper

### PR DESCRIPTION
**Pull request by ren0nie0**
This PR adds a data scraper for the counted data source. I also reorganized the files in the data scraper folder and intend to refactor existing notebooks to scripts.

### CODE REVIEW

**Comment by JMStudiosJoe** [comment]
these last conflicts are very small, you can likely just delete yours and push the changes to resolve these conflicts in the locks. When we re run the install we can have the correct lock files generated then

File: backend/scraper/data_scrapers/counted/scraper_data/README.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Please credit the Guardian as the source of this data in any work you do with it. Please link to http://www.theguardian.com/thecounted.
```
Comment (DMalone87) [inline backend/scraper/data_scrapers/counted/scraper_data/README.txt:1]: There should be a set of fields that indicate the source of this data as outlined in the [Incident Model](https://docs.google.com/document/d/1PCsRIKr0PF14Ul_qnem_UrUHz5RA0-5EmP5NIbDDQGA/edit?usp=sharing). Specifically:

- Publication Name
- Publication Date
- Author
- URL (Optional)

Let's try to get that information added with this.

File: backend/scraper/data_scrapers/counted/scraper_data/the-counted-2015.csv [hunk 2]
```diff
@@ -0,0 +1,1147 @@
+uid,name,age,gender,raceethnicity,month,day,year,streetaddress,city,state,classification,lawenforcementagency,armed
```
Comment (DMalone87) [inline backend/scraper/data_scrapers/counted/scraper_data/the-counted-2015.csv:1]: We may want to .ignore these... Over time, these sorts of files will blow up the repo.

**Review by JMStudiosJoe — APPROVED** [review]
lets resolve the conflicts together

**Review by JMStudiosJoe — APPROVED** [review]
fantastic

**Pull request merged**